CIFAR10/train/ajay_resnet9_1v100_pytorch.json

```diff
@@ -0,0 +1,25 @@
+{
+    "version": "v1.0",
+    "author": "Ajay Uppili Arasanipalai, Elliptigon",
+    "authorEmail": "ajay@elliptigon.com",
+    "framework": "pytorch 0.4.0",
+    "codeURL": "https://github.com/iyaja/cifar10-fast-2", 
```

Comment:
Please specify the commit hash either in the `codeURL` or as a separate `commitHash` field

Reply:
Done.

Reply:
@codyaustun I have made the changes you requested. Can you please review it one more time? What do I need to add/change?